jablib/src/main/java/org/jabref/model/metadata/UserHostInfo.java

```diff
@@ -0,0 +1,62 @@
+package org.jabref.model.metadata;
+
+import org.jspecify.annotations.NonNull;
+
+/**
+ * Record to represent user and host information.
+ * This is used to identify the user when retrieving file directories.
+ */
+public record UserHostInfo(@NonNull String user, @NonNull String host) {
```

Comment:
I think, all `@NonNull` annotations can be removed and `@NullMarked` added to this record?